### PR TITLE
Migrate to GA4

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Updates our Analytics scripts to use Google Analytics 4 (GA4) directly. Removes unneeded CSP policy values.

We previously had 2 env vars for GTM; `GOOGLE_TAG_MANAGER_ID` and `GOOGLE_TAG_MANAGER_AUTH`. Updated now so that use only use a single new env var `GOOGLE_TAG_ID`.

### How to review

Use this branch of runner with a staging environment - follow the instructions here: https://github.com/ONSdigital/eq-pipelines/pull/553

Check there are no CSP errors in the console.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by berroar** [comment]
[Remove GTM and migrate to GA4](https://trello.com/c/sBkUFzJI/5959-remove-gtm-and-migrate-to-ga4)

**Comment by berroar** [comment]
[Remove `unsafe-inline` from CSP policy (2 SP)](https://trello.com/c/YEVlBKnN/5847-remove-unsafe-inline-from-csp-policy-2-sp)

File: README.md [hunk 1]
```diff
@@ -347,15 +347,14 @@ The following environment variables must be set when deploying the app.
...
 | MIN_INSTANCES                | 1                | The minimum number of container instances that can be used for your Cloud Run service                          |
 | MAX_INSTANCES                | 1                | The maximum number of container instances that can be used for your Cloud Run service                          |
 | CPU                          | 4                | The number of CPUs to allocate for each Cloud Run container instance                                           |
 | MEMORY                       | 4G               | The amount of memory to allocate for each Cloud Run container instance                                         |
-| GOOGLE_TAG_MANAGER_ID        |                  | The Google Tag Manger ID - Specifies the GTM account                                                           |
-| GOOGLE_TAG_MANAGER_AUTH      |                  | The Google Tag Manger Auth - Ties the GTM container with the whole environment                                 |
+| GOOGLE_TAG_MANAGER_ID        |                  | The Google Tag ID - Specifies the GTM account                                                                  |
```
Comment (liamtoozer) [inline README.md:357]: v minor - `GOOGLE_TAG_ID` here as well?

Comment (berroar) [reply README.md]: Updated 👍

**Comment by berroar** [comment]
Ian Oliver has confirmed that the changes are working as expected

File: templates/layouts/_base.html [hunk 2]
```diff
@@ -73,32 +73,22 @@
...
-    and triggers can be set up in Google Tag Manager
-    -->
+  {% if google_tag_id %}
     <script nonce="{{ csp_nonce }}">
       dataLayer = {{ data_layer | tojson }}
     </script>
 
-    <!-- Google Tag Manager -->
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{ google_tag_id }}"></script>
```
Comment (MebinAbraham) [inline templates/layouts/_base.html:82]: This script will always be loaded even if they haven't accepted the tracking cookies. The last script 
 for GTM dynamically added this to the Document, we should try to do the same.

Comment (berroar) [reply templates/layouts/_base.html]: Updated now 👍

**Review by rmccar — APPROVED** [review]

File: templates/layouts/_base.html [hunk 3]
```diff
@@ -73,32 +73,36 @@
...
-    -->
+  {% if google_tag_id %}
     <script nonce="{{ csp_nonce }}">
       dataLayer = {{ data_layer | tojson }}
     </script>
 
-    <!-- Google Tag Manager -->
+    <!-- Google tag (gtag.js) -->
     <script nonce="{{ csp_nonce }}">
       function loadGTM() {
```
Comment (MebinAbraham) [inline templates/layouts/_base.html:83]: We should rename to `loadGoogleTagScript`

Comment (berroar) [reply templates/layouts/_base.html]: Updated 👍

**Review by MebinAbraham — APPROVED** [review]
LGTM

**Pull request merged**